src/views/Game/Game.tsx

```diff
@@ -2350,6 +2366,17 @@ export class Game extends React.PureComponent<GameProperties, GameState> {
     }
 
     cancelOrResign() {
+        let ratbag = false;
+
+        if (this.goban.engine.rengo && this.goban.engine.rengo_casual_mode) {
+            const team = this.goban.engine.rengo_teams.black.find(
+                (p) => p.id === data.get("user").id,
+            )
+                ? "black"
+                : "white";
+            ratbag = this.goban.engine.rengo_teams[team].length > 1;
+        }
+        console.log("ratbag: ", ratbag);
```

Comment:
Might should remove this before production :grin: might should change the variable name too, it's amusing, but boring code is more maintainable and we'll get less curses put upon us by future developers.